components/__design__/logos.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const JSLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
```

Comment:
I wonder if we need `<div>` 2 times? Maybe only one div container is enough?

Reply:
Good point, just remove the unnecessary <div> here